src/App.js

```diff
@@ -73,11 +56,8 @@ function App (){
           <div className='header-image' style={{backgroundImage:`url(${iconFile})`
           }}
           />
-
           <div>{htmlHeader}</div>
-        </div>
-        
-            
+        </div>   
           <div className="arrow-up"></div>
```

Comment:
If there is no content inside <div></div>
it is better to code like this:
`<div className="arrow-up" />`